bw2io/backup.py

```diff
@@ -74,7 +144,26 @@ def get_project_name(fp):
     project_name = get_project_name(fp)
 
     with tarfile.open(fp, "r|gz") as tar:
-        tar.extractall(projects._base_data_dir)
```

Comment:
You seem to have included commits by other contributors in your pull request:

<img width="920" alt="Screenshot 2023-03-15 at 12 39 50" src="https://user-images.githubusercontent.com/23102087/225298502-76db4f8d-2928-4123-9d72-dd85c7c0781d.png">


You MUST ensure that your pull requests contains ONLY your own edits to the docstrings. Feel free to delete this PR and create a new one (with only your own contributions.